concurrent_test.go

```diff
@@ -285,18 +365,41 @@ func formatBytes(val []byte) string {
 	return hex.EncodeToString(val)
 }
 
-func saveDataIfFailed(t *testing.T, db *btesting.DB) {
+/*
+*********************************************************
+Functions for persisting test data, including db file
+and operation history
+*********************************************************
+*/
```

Comment:
Please consider using standard godoc format https://go.dev/blog/godoc

Or better, don't comment and focus on function naming. This function has a good name and have only couple of logic lines. `saveDataIfFailed` is great name.

Reply:
The comment isn't for a specific function or method, it's for a block of source code.  Note that I am planing to move them to separate files